runtime/ts/storage/firebase-storage.ts

```diff
@@ -634,7 +634,7 @@ class FirebaseCollection extends FirebaseStorageProvider {
   private readonly initialized: Promise<void>;
   private pendingWrites: {value: {}, storageKey: string}[] = [];
   private resolveInitialized: () => void;
-  private localId = 0;
+  private localKeyId = Date.now();
```

Comment:
Unsure how this is intended to work, but can we use  `Math.floor(Math.random()*1e13))` instead to make it clear that this is random and not supposed to be tied to the date?